src/DataTable/README.md

```diff
@@ -275,20 +275,27 @@ To enable proper selection behavior with backend pagination (i.e., when ``isSele
         },
       ]}
       bulkActions={[
-        {
-          buttonText: 'Download CSV',
-          handleClick: (data) => console.log('Download CSV', data),
-        },
-        // custom button function that utilizes clearSelection function provided by the table instance
-        ({tableInstance})=>{
-          return {
-            buttonText: `Clear Selection`,
-            handleClick: () => {
-              console.log('Clear selection');
-              tableInstance.clearSelection()
+        ({ as, selectedFlatRows }) => React.createElement(
```

Comment:
Thank you for the cool idea with `cloneElement`. I found out that I can get `as` from `action.component.props.as` that probably the most suitable solution for the `TableActions/BulkActions`, because we can do next:
```
<EnrollAction /> // don't pass `as`, `Button` is default
<EnrollAction as={Button} /> // `as` explicitly that equals to the first one
<EnrollAction as={SomethingElse} />
```
Examples are also updated with the new versions.